src/utils/player.js

```diff
@@ -1,10 +1,12 @@
 import nanoid from "nanoid";
 import defaultAvatar from "../images/dragon.png";
 
-export function Player(name, score, avatar) {
+export function Player(name, sex, level, bonus, avatar) {
```

Comment:
Same comment as above, I'd refactor this to accept an object with these values instead of arguments!